src/wallet/rpcdump.cpp

```diff
@@ -35,7 +35,7 @@ std::string static EncodeDumpString(const std::string &str) {
     std::stringstream ret;
     for (const unsigned char c : str) {
         if (c <= 32 || c >= 128 || c == '%') {
-            ret << '%' << HexStr(Span<const unsigned char>(&c, 1));
```

Comment:
If anyone is confused about the bot marking this as needing a rebase, and then the merge. The conflict was in `src/wallet/rpcdump.cpp`. The conflicting code was moved from there, to `src/wallet/rpc/backup.cpp` in #23647. You can see `src/wallet/rpc/backup.cpp` being updated in the merge commit: https://github.com/bitcoin/bitcoin/commit/8b1de78577a02300340cec37e26cd6418ef6e645#diff-c4eeb9bd8e84ac276465d100d6ac4c1c9b9ddc145827a03dfe11932598b028b9R38.

Reply:
yeah, daily reminder to stop trusting GitHub metadata